esp/scm/ws_workunits.ecm

```diff
@@ -1718,6 +1719,10 @@ ESPrequest [nil_remove] WUGetStatsRequest
     unsigned MaxScopeDepth;
     boolean IncludeGraphs;
     boolean CreateDescriptions;
+    [min_ver("1.62")] int64 MinValue; // Should be uint64 but esdl does not support it.
+    [min_ver("1.62")] int64 MaxValue;
+    [min_ver("1.62")] string Filter; // arbitrary text filter
+    //MORE: Some indication of which fields need to be returned - to reduce the amount of data returned.  Is there a general esp mechanism?
```

Comment:
I don't think there isn't a general mechanism that is relevant here.  There is something more relevant to interface management and versioning, but not run time field selection.